client/src/containers/home/StatisticFragment.js

```diff
@@ -5,31 +5,29 @@ import { Trans } from '@lingui/macro';
 import { ReactComponent as PeopleRisk } from '../../images/user-solid.svg';
 import { ReactComponent as EvaluatedBuildings } from '../../images/house-damage-solid.svg';
 import { ReactComponent as ConsolidatedBuildings } from '../../images/home-solid.svg';
```

Comment:
In the new version of the specs, the new icon is a wrench. [There is one in font awesome](https://fontawesome.com/icons/wrench), though it should be flipped horizontally